src/libraries/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -899,6 +892,11 @@ private static int GetWaitStateReferenceCount(object waitState)
         [DllImport("libc")]
         private static extern int chmod(string path, int mode);
 
+        private static void ChMod(string filename, string mode)
+        {
+            Assert.Equal(0, chmod(filename, Convert.ToInt32(mode, 8)));
```

Comment:
Nit: a recommended way is:
```suggestion
    int returnCode;
    do
    {
        returnCode = chmod(filename, Convert.ToInt32(mode, 8)));
    }
    while (returnCode == -1 && Marshal.GetLastWin32Error() == EINTR);
    
    Assert.Equal(0, returnCode);
```

Reply:
Thanks for the suggestion. Since this `Assert` never failed so far, I'm going to keep it as it was.